src/components/sidebar/components/sidebar-back-to-link/index.tsx

```diff
@@ -26,7 +33,7 @@ const SidebarBackToLink: React.FC<{
   )
 }
 
-const PlaceholderBackToLink: React.FC = () => {
+const PlaceholderBackToLink = (): ReactElement => {
```

Comment:
Made this update to more accurately reflect what's happening. `React.FC` means that the `children` prop is passed to the component, but that's not happening here.